OutOfSchool/OutOfSchool.WebApi/Common/GettingUserProperties.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Security.Authentication;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Http;
+using OutOfSchool.Common;
+using OutOfSchool.Common.Extensions;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.WebApi.Common
+{
+    public static class GettingUserProperties
```

Comment:
Please cover this class methods with tests

Reply:
Could you, please, give us some examples of tests or ideas what we should test here? Sorry, but we can't catch the main idea of testing these one row methods.

Reply:
At least you could check methods pre-, post- conditions. Obvious cases are - check for null and throwing exception.

I.e.
```
Assert.IsNull(() => GetUserId((ClaimsPrincipal)null)); 
Assert.Throws<AuthenticationException>(() => GetUserId((HttpContext)null)); 
```

If it's possible to mock HttpContext, ClaimsPrincipal (and here you see necessity to use related interfaces)
you could mock them and check not-null cases.
 
Assert.AreEqual(mockedUser, GetUserId(mockedHttpContext));

Reply:
Done